include/sm64.h

```diff
@@ -70,6 +70,8 @@ enum RenderLayers {
     LAYER_OCCLUDE_SILHOUETTE_OPAQUE,
     LAYER_OCCLUDE_SILHOUETTE_ALPHA,
 #endif
+    LAYER_CIRCLE_SHADOW,
```

Comment:
This should be after `LAYER_TRANSPARENT_DECAL`, because more often than not you're going to want your shadows on top of decals, not the reverse.